dc-cudami-editor/src/components/modals/IframeAdderModal.jsx

```diff
@@ -21,7 +21,12 @@ class IframeAdderModal extends Component {
       title: '',
       width: '',
     }
-    subscribe('editor.show-iframe-modal', () => {
+    subscribe('editor.show-iframe-modal', (_msg, data = {}) => {
+      this.setState({
+        ...this.state,
```

Comment:
das `...this.state` muss man nicht machen, React ist selber schlau genug dass es bei partiellen State-Updates nicht die fehlenden Keys mit `undefined` überschreibt :-)

> You may optionally pass an object as the first argument to setState() instead of a function:
> ```
> setState(stateChange[, callback])
> ```
> This performs a *shallow merge* of stateChange into the new state, e.g., to adjust a shopping cart item quantity:
> ```
> this.setState({quantity: 2})
> ```
> https://reactjs.org/docs/react-component.html#setstate

Reply:
Hast vollkommen recht, hab ich entfernt :)